### DISCUSSION TITLE
Application for GitHub Education

### DISCUSSION THREAD

**Discussion by ProHarshit** [community/community#38636] [GitHub Education] [answered]
If I am 13 years old and in class 8,then can I apply for GitHub Student Developer Pack?

**Comment by ProHarshit** [comment, marked as answer]
Thank you GitHub .
Thank you very much
I got it approved using my official school id.
And I am 13 years old and in class 8
Thank you once again

**Comment by samus-aran** [comment]
We're so happy to hear that you've been verified @ProHarshit and excited to be part of Global Campus! May you have a fantastic journey in open source with GitHub 🎉
